aws_lambdas/twilio_webhook.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+import os
+from typing import Any, Dict
+from urllib.parse import unquote_plus
+
+import boto3
+from twilio.request_validator import RequestValidator
+
+from stopcovid.dialog.command_stream.publish import CommandPublisher
+from stopcovid.utils.logging import configure_logging
+
+configure_logging()
+
+
+def handler(event, context):
+    kinesis = boto3.client("kinesis")
+    stage = os.environ["STAGE"]
+
+    form = extract_form(event)
+    if not is_signature_valid(event, form, stage):
+        logging.warning("signature validation failed")
+        return {"statusCode": 403}
+    if "MessageStatus" in form:
+        logging.info(f"Outbound message to {form['To']}: Recording STATUS_UPDATE in message log")
+        kinesis.put_record(
+            Data=json.dumps({"type": "STATUS_UPDATE", "payload": form}),
+            PartitionKey=form["To"],
+            StreamName=f"message-log-{stage}",
+        )
+        return
+
+    logging.info(f"Inbound message from {form['From']}: Recording INBOUND_SMS in message log")
+    CommandPublisher().publish_process_sms_command(form["From"], form["Body"])
+    kinesis.put_record(
+        Data=json.dumps({"type": "INBOUND_SMS", "payload": form}),
+        PartitionKey=form["From"],
```

Comment:
For my own sanity:

- Inbound sms events are published after twilio receives a message from a user 
- Inbound message means that our system is receiving a message From a user
- We should be using the "From" phone number
- This is correct